mllib/src/main/scala/org/apache/spark/ml/feature/RFormula.scala

```diff
@@ -105,12 +112,14 @@ private[feature] trait RFormulaBase extends HasFeaturesCol with HasLabelCol with
    * @group param
    */
   @Since("2.3.0")
-  final val stringIndexerOrderType: Param[String] = new Param(this, "stringIndexerOrderType",
+  final val stringIndexerOrderType: Param[String] = new Param(
+    this,
+    "stringIndexerOrderType",
     "How to order categories of a string FEATURE column used by StringIndexer. " +
-    "The last category after ordering is dropped when encoding strings. " +
-    s"Supported options: ${StringIndexer.supportedStringOrderType.mkString(", ")}. " +
-    "The default value is 'frequencyDesc'. When the ordering is set to 'alphabetDesc', " +
-    "RFormula drops the same category as R when encoding strings.",
+      "The last category after ordering is dropped when encoding strings. " +
+      s"Supported options: ${StringIndexer.supportedStringOrderType.mkString(", ")}. " +
+      "The default value is 'frequencyDesc'. When the ordering is set to 'alphabetDesc', " +
+      "RFormula drops the same category as R when encoding strings.",
```

Comment:
The changes here and above don't seem directly relevant unless I'm missing something. I wouldn't reformat these.

Reply:
reverted format changes.